testing/novnc/alpine-specific-launch.js.patch

```diff
@@ -0,0 +1,17 @@
+--- a/utils/launch.sh
++++ b/utils/launch.sh
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env bash
++#!/usr/bin/env ash
+ 
+ # Copyright 2016 Joel Martin
+ # Copyright 2016 Solly Ross
+@@ -141,7 +141,7 @@
+ ${WEBSOCKIFY} --web ${WEB} ${CERT:+--cert ${CERT}} ${PORT} ${VNC_DEST} &
+ proxy_pid="$!"
+ sleep 1
+-if ! ps -p ${proxy_pid} >/dev/null; then
++if ! ps | grep ${proxy_pid} >/dev/null; then
```

Comment:
This is _very_ wrong. `ps` without arguments prints pid, user, time and command. Your grep matches any line that contains `$proxy_pid` number; if `proxy_pid=42`, then it will match even processes with pid `142`, time `0:42` or with `42` on command path.

Correct solution is `[ -z "$proxy_pid" ] || ! ps -o pid= | grep -qw ${proxy_pid}`.